app/scripts/blocker_afd.js

```diff
@@ -0,0 +1,10 @@
+document.body.innerHTML = `
```

Comment:
`innerHTML` is not allowed due to security reasons:
https://wiki.mozilla.org/Add-ons/Reviewers/Guide/Reviewing#Step_2:_Automatic_validation

You could use `addBlocker` from `blocker_general`